package.json

```diff
@@ -46,5 +46,9 @@
     "http",
     "request-http",
     "debug"
-  ]
+  ],
+  "jest": {
+    "verbose": true,
+    "testURL": "http://localhost/"
```

Comment:
Otherwise I get "SecurityError: localStorage is not available for opaque origins"